### PR TITLE
Basic React components

### PR DESCRIPTION
**Pull request by simona1**
Very basic React components that pass the shallow rendering tests, lint and flow checks.

### CODE REVIEW

File: src/Toolbar.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import 'semantic-ui-css/semantic.min.css';
```
Comment (HamidAghdaee) [inline src/Toolbar.jsx]: dont need this. the css can be loaded like this in index.html:
<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.11/semantic.min.css"></link>

File: package.json [hunk 2]
```diff
@@ -3,23 +3,37 @@
...
+    "enzyme-to-json": "1.5.1",
+    "flow": "0.2.3",
+    "flow-bin": "0.51.1",
+    "jest": "20.0.4",
+    "jest-cli": "20.0.4",
+    "prop-types": "15.5.10",
+    "react": "15.6.1",
+    "react-dom": "15.6.1",
+    "react-fontawesome": "1.6.1",
+    "semantic-ui-css": "2.2.11",
```
Comment (HamidAghdaee) [inline package.json]: no need for semantic-ui-css

File: src/MessageList.jsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
+// @flow
+
```
Comment (HamidAghdaee) [inline src/MessageList.jsx:2]: This will surely be broken up into smaller components. You can do it in this PR or later.

File: package.json (hunk 2 above)
Comment (simona1) [reply package.json]: Done

File: src/Toolbar.jsx (hunk 1 above)
Comment (simona1) [reply src/Toolbar.jsx]: Done

**Pull request merged**